webcompat/webhooks/helpers.py

```diff
@@ -49,7 +50,30 @@ def set_label(label, issue_number):
     api_post('labels', payload, issue_number)
 
 
-def dump_to_db(title, body, issue_number):
+def extract_domain_name(url):
```

Comment:
We have a similar method here: https://github.com/webcompat/webcompat.com/blob/d40acd13eb7956f34c7e087f8ae3a1439bb19511/webcompat/form.py#L143

I haven't looked closely, but can we consolidate? Is there some case that this one handles that the existing one doesn't?